docs/content/preview/secure/authentication/ldap-authentication-ysql.md

```diff
@@ -62,17 +62,17 @@ The configurations supported for simple bind mode.
 
 In `Search + Bind` mode, YB-Tserver will bind to the LDAP directory with a fixed username and password, specified with `ldapbinddn` and `ldapbindpasswd`, and performs a search for the user trying to log into the database. This mode is commonly used by LDAP authentication schemes in other software.
```

Comment:
In `Search + Bind` mode --- Why is it wrapped in code? It was search+bind mode in bold earlier. Should be just plain text, no?